ceaos/sensors/camera_sensor.py

```diff
@@ -0,0 +1,60 @@
+"""
+This file contains a class for a camera sensor that encodes image data
+"""
+import base64
+from sensor_definition import Sensor  
+
+class Camera(Sensor):
```

Comment:
You should call this an artificial camera, so the name is available for live cameras. Based on my reading, an image is set, and then returned upon each call to read_sensor.